lambda/src/main/scala/pricemigrationengine/model/CohortItem.scala

```diff
@@ -8,24 +8,53 @@ import java.time.{Instant, LocalDate}
 case class CohortItem(
     subscriptionName: String,
     processingStage: CohortTableFilter,
-    amendmentEffectiveDate: Option[LocalDate] = None,
     currency: Option[Currency] = None,
+
+    // Pre migration price
     oldPrice: Option[BigDecimal] = None,
+
+    // Price derived from the Estimation step, without capping
     estimatedNewPrice: Option[BigDecimal] = None,
+
+    // Price (with possible capping) used in the communication to the user and sent to Salesforce
+    commsPrice: Option[BigDecimal] = None,
+
+    // Price read from the post amendment subscription
+    newPrice: Option[BigDecimal] = None,
+
+    //
```

Comment:
no comment?

Reply:
Yeah. It's to force a separation (stylistic presentation of the code). Otherwise the linter (scalafmt) remove the space :)